compel/plugins/std/log.c

```diff
@@ -355,6 +355,27 @@ void print_on_level(unsigned int loglevel, const char *format, ...)
 	sbuf_log_flush(&b);
 }
 
+int flog_encode_msg(int loglevel, unsigned int nargs, unsigned int mask, const char *format, ...)
```

Comment:
Probably copying print_on_level code here is not a good thing.

Reply:
Without including that function, I received an error: external symbol in PIE. I had copied the print_on_level code as I didn’t want to lose any log messages. I am also not sure if we would include binary logging in compel or if we would be choosing some other logging technique for it. 
 
Rather than copying the print_on_level code, should I make it return without printing, or should I make both the functions point to a goto label, or is there some other alternative that I could use to solve this?